pkgs/development/python-modules/lox/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+
+  pathos,
+  pytestCheckHook,
+  pytest-mock,
+  setuptools,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "lox";
+  version = "0.12.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "BrianPugh";
+    repo = "lox";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Iv3ZdfsvFLU6lhlH1n+eQ+KIrXESsnC1S2lVFnKFV08=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ pathos ];
+
+  pythonRemoveDeps = [ "sphinx-rtd-theme" ];
+
+  # setup.py requires pytest-runner for setuptools, which is wrong
+  postPatch = ''
+    substituteInPlace setup.py --replace '"pytest-runner",' ""
```

Comment:
```suggestion
    substituteInPlace setup.py --replace-fail '"pytest-runner",' ""
```

Reply:
I've updated it, but I'm curious as to why the prejudice against the existing replace. Since I'm simply removing a value that shouldn't be present, why do we want this to fail if upstream should remove it already?

Reply:
That means we have to update our Nix code. This can even be considered, to some extent, as a security issue in case something fails while there are has been no version bump.